shipment/buyer/receive/README.md

```diff
@@ -19,7 +19,7 @@ Contact support if you need the polling API to receive shipment messages:
 {% page-ref page="../../api/webhook-vs-polling.md" %}
 
 {% hint style="warning" %}
-This shipment module is under development. The API and documentation may change.
+The shipment module is under development. The API and documentation may change.
```

Comment:
Don't we consider this final already?

Reply:
The MVP is in pilot phase.
Lets remove it when the customer is live with shipments.